contracts/src/interfaces/IRewardClaim.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+interface IRewardClaim {
+    /// @notice Proof for rewards merkle tree. Obtained from Espresso query service API.
+    struct LifetimeRewardsProof {
+        bytes32[] siblings;
+    }
+
+    /// @notice User claimed rewards
+    event RewardClaimed(address indexed user, uint256 amount);
```

Comment:
```suggestion
    event RewardsClaimed(address indexed user, uint256 amount);
```

Probably a bit more consistent to use the plural here too.

Reply:
https://github.com/EspressoSystems/espresso-network/pull/3610/commits/39ee5deb0a474f183b0d1678eea9af6ff9e10243